docs/taskbackground.rst

```diff
@@ -35,23 +67,197 @@ The tree of the above event looks something like
+The below table gives an overview of the observables and their binning.
+
+.. list-table:: Histogram Observables
+   :widths: 25 15 30 30
+   :header-rows: 1
+
+   * - Region
+     - Name
+     - Binning (Processing Step)
+     - Binning (Modelling Step)
+   * - 4j2b (Control Region)
+     - :math:`H_T`
+     - 25 bins from 50 to 550
+     - 11 bins from 50 to 550
+   * - 4j1b (Signal Region)
+     - :math:`m_{bjj}`
+     - 25 bins from 50 to 550
+     - 11 bins from 50 to 550
+
+
+**4j2b (Control Region)**: Event :math:`H_T`
+
+In the control region, we require there to be at least two b-tagged jets (``btagCSVV2 > B_TAG_THRESHOLD``:sup:`*`) and at least four jets.
+
+The observable in this region is the event :math:`H_T`, which is the sum of the :math:`p_T` of all jets in each event.
+
+
+**4j1b (Signal Region)**: :math:`m_{bjj}` (Reconstructed top mass)
+
+In the signal region, we require there to be exactly one b-tagged jet (``btagCSVV2 > B_TAG_THRESHOLD``:sup:`*`) and at least four jets.
+
+To measure the :math:`t\bar{t}` cross-section, we use an observable that approximately reconstructs the top quark mass.
+This is done using the following steps:
+
+#. Filter the events using the criteria explained above (including requiring at least four jets and exactly one lepton)
+#. Calculate all possible combinations of three jets from all jets in each event (for example, in our implementation we use ``ak.combinations``)
+#. Ensure that there is at least one b-tagged jet 
+#. Choose the combination with the largest combined transverse momentum (:math:`p_T`)
+#. Calculate combined mass of trijet system
+
+:sup:`*` **NOTE:** All jets have an associated ``btagCSVV2`` value, ranging from 0 to 1, which quantifies how likely the jet is come from a b quark. We utilize a threshold value, ``B_TAG_THRESHOLD=0.5``, to accept a jet as "b-tagged".
+
+Successful top mass reconstruction can be inferred from the below histogram, which is peaked around the top mass (:math:`\approx 173 GeV`). 
+We can also see that our selection criteria worked, since the majority of events in the histogram are from the :math:`t\bar{t}` sample.
+
+.. image:: images/analysis.png
+  :width: 80%
+  :alt: Top mass distribution demonstrating successful top mass reconstruction.
+
+
+5. Systematic Uncertainties
 ---------------------------------------------------------------
-To measure the :math:`t\bar{t}` cross-section, we use an observable that approximately reconstructs the top quark mass. We do this in two different ways. The first uses no machine learning. Within an event, the trijet system with the highest transverse momentum (:math:`p_T`) is selected. We then calculate the combined mass of these three jets.
+This section explains how to calculate the various systematic uncertainties we incorporate into our model. Some of these affect the kinematics of the event, so the observables will need to be calculated for each variation.
+
+.. list-table:: Systematic Uncertainties
```

Comment:
There are a few additional ones we need that function differently: luminosity (which doesn't technically need histograms, as i just changes normalization of all samples) and the three ttbar variations where the nominal sample is replaced by other samples.

Reply:
I added these... although I am not confident in my descriptions, so please check me.